subtitles.go

```diff
@@ -230,18 +281,27 @@ type StyleAttributes struct {
 	WebVTTVertical       string
 	WebVTTViewportAnchor string
 	WebVTTWidth          string
+	WebVTTColor	     string
 }
 
 func (sa *StyleAttributes) propagateSSAAttributes() {}
 
 func (sa *StyleAttributes) propagateSTLAttributes() {}
 
 func (sa *StyleAttributes) propagateTeletextAttributes() {
-	if sa.TeletextColor != nil {
+	if sa.TeletextColor != nil {		
 		sa.TTMLColor = "#" + sa.TeletextColor.TTMLString()
+		sa.WebVTTColor=sa.TeletextColor.WebVTTColor()				
 	}
 }
 
+func (sa *StyleAttributes) propagateTeletextPositionJustification() {		
```

Comment:
As mentioned earlier, I'd rather use the `propagateSTLAttributes` method for that